docs/concepts/true-hybrid-search.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: "True Hybrid search"
```

Comment:
imo this isn't really a guide since both semantic search guide and keyword search guide show hybrid. this is page is in the same category as architecture or similar. we should highlight it more

Reply:
to really nail this point, we should add actual comparisons to other dbs and call them out for being imposters. for example, pinecone only has hybrid in the python client (bc its implemented client-side). tpuf requires you to write bunch of code.

since this is a differentiator, we need to actaully convince the user topk is better